src/pages/Home/Home.js

```diff
@@ -98,9 +117,27 @@ class Home extends React.Component {
                           />
                         </StyledLink>
                         <StyledLink to={`/user/${value.user.username}`}>
-                          <UserName username={value.user.name} />
+                          <div>{value.user.name}</div>
                         </StyledLink>
-                        <Icon icon={<FaHeart />} stats={value.likes} />
+                        {Object.values(this.state.favoritePhotos).find((id) => {
+                          return id === value.id;
+                        }) ? (
```

Comment:
but if `this.state.favoritePhotos` is an object, you don't need to convert it to an array. You can just do `this.state.favoritePhotos[value.id]`